### PR TITLE
Use Dart Sass for Sass implementation

### PR DESCRIPTION
**Pull request by andygout**
Fixes https://github.com/Financial-Times/dotcom-page-kit/issues/678.

Switches from using LibSass to Dart Sass for Sass implementation.

Work was validated by running `npm run build` in `./examples/ft-ui` having added the below lines to its `/client/main.scss` file.

```scss
@if true {
  $not-already-defined: true !global;
}
```

Dart Sass throws a warning if a global variable is defined outside of global scope, which LibSass doesn't.

To ensure warnings are enabled you can add to the `main.scss` file the below line which outputs a warning.

```scss
@warn 'warning text';
```

#### Before (i.e. LibSass)
before

---

#### Before (i.e. Dart Sass)
after

#### TODO:
- Testing of this Page Kit branch with consuming apps to ensure that `n-` components are spec compliant.

- [x] `next-static`: https://github.com/Financial-Times/next-static/pull/134. 
- [x] `next-search-page`: no changes required.
- [x] `next-video-page`: no changes required.
- [x] `next-errors`: no changes required.
- [x] `next-epaper`: no changes required.
- [x] `next-tour-page`: https://github.com/Financial-Times/next-tour-page/pull/261.
- [x] `next-graphics-page`: https://github.com/Financial-Times/next-graphics-page/pull/249.
- [x] `next-profile`: https://github.com/Financial-Times/next-profile/pull/292.
- [x] `next-syn-list`: no changes required.
- [x] `next-product`: https://github.com/Financial-Times/next-product/pull/867.
- [x] `next-b2b-prospect`: no changes required.
- [x] `next-retention`: no changes required.
- [x] `next-subscribe`: no changes required.
- [x] `next-stream-page`: no changes required.
- [x] `next-front-page`: no changes required.
- [x] `next-myft-page`: no changes required.
- [x] `next-article`: https://github.com/Financial-Times/next-article/pull/3691.
- [x] `next-control-centre`: no changes required.

#### References:
- [Sass: Dart Sass](https://sass-lang.com/dart-sass).
- [GitHub: webpack-contrib/sass-loader - Options](https://github.com/webpack-contrib/sass-loader#user-content-options).

### CODE REVIEW

File: packages/dotcom-build-sass/package.json [hunk 1]
```diff
@@ -25,9 +25,10 @@
     "css-loader": "^3.0.0",
     "cssnano": "^4.1.10",
     "dlv": "^1.1.3",
+    "fibers": "^4.0.2",
```
Comment (i-like-robots) [inline packages/dotcom-build-sass/package.json]: Is this required or does it just help to speed things up?

Comment (andygout) [reply packages/dotcom-build-sass/package.json]: The latter. It is detailed in the second of the links in the PR description.

Comment (i-like-robots) [reply packages/dotcom-build-sass/package.json]: Does it seem to be effective?

Comment (andygout) [reply packages/dotcom-build-sass/package.json]: So far I have only used it to build the `ft-ui` example which does not have much to build so I will find out when I start testing it on Page Kit-consuming apps.

Comment (andygout) [reply packages/dotcom-build-sass/package.json]: Testing with `fibers` for `next-myft-page` build:
```
real	0m20.085s
user	0m22.182s
sys	0m1.758s
```

Testing without `fibers` for `next-myft-page` build:
```
real	0m18.221s
user	0m21.776s
sys	0m1.603s
```

🤷‍♂ I will remove it.

Comment (andygout) [reply packages/dotcom-build-sass/package.json]: Removed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/764/commits/e062b2c147733d6354a2edbee31d750a17bbc404).

**Pull request merged**